models/template_helpers.go

```diff
@@ -193,7 +194,7 @@ func RemoveTemplate(q *reform.Querier, name string) error {
 	}
 
 	if inUse {
-		return status.Errorf(codes.FailedPrecondition, "Failed to delete rule template %s, as it is being used by some rule.", name)
+		return status.Errorf(codes.FailedPrecondition, `You can't delete a rule template "%s" when it's being used by a rule.`, template.Summary)
```

Comment:
AFIK It should be `the` instead of `a`, because we are talking here about one particular rule template, we even name it. 
```suggestion
		return status.Errorf(codes.FailedPrecondition, `You can't delete the rule template "%s" when it's being used by a rule.`, template.Summary)
```

Reply:
Probably it should be even like this:
`You can't delete the "%s" rule template...`